share/spice/twitch_streams/twitch_streams.js

```diff
@@ -0,0 +1,69 @@
+(function (env) {
+    "use strict";
+    env.ddg_spice_twitch_streams = function(api_result){
+
+        if (api_result.error) {
```

Comment:
You'll want to update this to include checking if `api_result` exists 
`(!api_result || api_result.error) {`